src/helpers/build.js

```diff
@@ -26,6 +26,7 @@ const concat = require('concat')
 const os = require('os')
 
 const spinner = require('./spinner')
+const isInstalledGlobally = require('is-installed-globally')
```

Comment:
Can we move this import one up? I prefer to put all 'package' import on top, followed by relative imports.